config/ini.go

```diff
@@ -184,6 +184,8 @@ func (ini *IniConfig) ParseData(data []byte) (Configer, error) {
 	if err := ioutil.WriteFile(tmpName, data, 0655); err != nil {
 		return nil, err
 	}
+	defer os.Remove(tmpName)
```

Comment:
@fugr I think it would be better if you handle the returned `error` from `os.Remove(tmpName)`:
```
defer func() {
     if err := os.Remove(tmpName); err != nil {
            return nil, err
     }
}()
```